### PR TITLE
#13234: Support clip, clamp with min/max tensor

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
#13234
**Work done** :  Provide support for min, max as tensor in clamp, clip
https://github.com/tenstorrent/tt-metal/actions/runs/11665096589 - PASSED

### CODE REVIEW

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp [hunk 1]
```diff
@@ -82,6 +96,7 @@ void bind_unary_composite_optional_floats_with_default(
             py::arg("memory_config") = std::nullopt});
 }
 
+
```
Comment (Aswinmcw) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:99]: Remove

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by umadevimcw — APPROVED** [review]

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp [hunk 2]
```diff
@@ -67,6 +67,20 @@ void bind_unary_composite_optional_floats_with_default(
         module,
         operation,
         doc,
```
Comment (mcw-anasuya) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:69]: Can you add supported parameters as well?

Comment (VirdhatchaniKN) [reply ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:74]: Screenshot 2024-11-05 at 9 08 25 AM

**Review by eyonland — APPROVED** [review]

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp (hunk 2 above)
Comment (VirdhatchaniKN) [reply ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:74]: Screenshot 2024-11-05 at 9 09 07 AM

**Review by mcw-anasuya — APPROVED** [review]

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/11677098461 - passed

**Pull request merged**